pkg/configtx/msp.go

```diff
@@ -607,6 +633,54 @@ func pemEncodePKCS8PrivateKey(priv crypto.PrivateKey) ([]byte, error) {
 	return pem.EncodeToMemory(&pem.Block{Type: "PRIVATE KEY", Bytes: privBytes}), nil
 }
 
+// UpdateConsortiumMSP updates the MSP config for the specified consortium org group.
+func (c *ConfigTx) UpdateConsortiumMSP(updatedMSP MSP, consortiumName string, orgName string) error {
```

Comment:
We've decided to standardize on naming these `SetConsortiumMSP`, also this should probably be `SetConsortiumOrgMSP` but we use the same naming convention for `SetOrdererMSP` and `SetApplicationMSP` so I'm fine leaving it as `SetConsortiumMSP`.

Reply:
Yeah, I have replaced Update with Set for the three functions.